app/src/main/java/vn/mbm/phimp/me/editor/editimage/EditImageActivity.java

```diff
@@ -189,7 +197,9 @@ public void onClick(View v) {
         mRotateFragment = RotateFragment.newInstance();
         mAddTextFragment = AddTextFragment.newInstance();
         mPaintFragment = PaintFragment.newInstance();
-
+        banner.setBackgroundColor(themeHelper.getPrimaryColor());
```

Comment:
It should be write in onResume if in case user change the theme and this activity is in the pause state at that time.